android-core/src/main/java/com/mparticle/Widget.java

```diff
@@ -0,0 +1,20 @@
+package com.mparticle;
```

Comment:
Was on a call with SYD and they suggested we align this to be called `RoktEmbeddedView` instead of `Widget` so that we standardize the Rokt iOS/Android kits more.  Please confirm @sahil-rokt

Reply:
Yes, we agreed to use the common in both platform called as `RoktEmbeddedView`